modules/installation-configure-proxy.adoc

```diff
@@ -70,6 +70,9 @@ The `Proxy` object `status.noProxy` field is populated with the values of the `n
 
 For installations on Amazon Web Services (AWS), Google Cloud Platform (GCP), Microsoft Azure, and {rh-openstack-first}, the `Proxy` object `status.noProxy` field is also populated with the instance metadata endpoint (`169.254.169.254`).
 ====
+* If your cluster is on AWS, you added the `ec2.<region>.amazonaws.com` and
+`elasticloadbalancing.<region>.amazonaws.com` endpoints to your VPC endpoint. Because the proxy works only on container level, not the node level, you must include these endpoints in your VPC in addition to adding the public IP address of the EC2 API to your allowlist.
```

Comment:
missing s3 endpoint `s3.<region>.amazonaws.com` here?

there are 3 endpoints in the above section:

> If you are working in a disconnected environment or use a proxy, you are unable to reach the public IP addresses for EC2 and ELB endpoints. To resolve this, you must create a VPC endpoint and attach it to the subnet that the clusters are using. Create the following endpoints:
> 
> * `ec2.<region>.amazonaws.com`
> * `elasticloadbalancing.<region>.amazonaws.com`
> * `s3.<region>.amazonaws.com`